.github/workflows/ui-tests.yaml

```diff
@@ -14,6 +14,8 @@ on:
     branches:
       - "main"
       - 'release-**'
+  schedule:
```

Comment:
I don't know how the event structure in the GitHub API is, but are we sure, that `if: github.event.pull_request.draft == false` will not also apply for the schedule trigger and therefore trigger `run-ui-tests` and `run-ui-tests-periodic`?

Reply:
I've just separated two workflows, since in 2nd look it looks cleaner this way.

Reply:
Ok, I think this might be an input for our team goal. It might be better to have one workflow file for all scheduled jobs.